pkg/dpll-netlink/dpll.go

```diff
@@ -154,6 +155,8 @@ func ParseDeviceReplies(msgs []genetlink.Message) ([]*DoDeviceGetReply, error) {
 				reply.ClockId = ad.Uint64()
 			case DPLL_A_TYPE:
 				reply.Type = ad.Uint32()
+			default:
+				log.Println("default", ad.Type(), len(ad.Bytes()), ad.Bytes())
```

Comment:
If in the future API will change and we will fail to parse it, we'll see it in the logs much clearer than "netlink: attribute 5 is not a uint32; length: 8" as it was before this commit